### ISSUE TITLE
PokemonsNotToTransfer list not observed by bot

### ISSUE THREAD

**Issue by popsoda27** [NoxxDev/NecroBot#685]
## Stop! Before you create this issue (you can delete this section when opening the issue):
1. Have you validated that your config/auth.json and config/config.json is valid JSON? Use http://jsonlint.com/ to check.
2. Have you searched to see if there are other issues for the same issue? If so, comment on that issue instead.
3. If your question is referring to how to set up or use the bot, please join our discord instead of posting an issue: https://discord.gg/VXKxNFr

### Expected Behavior
To not transfer pokemon listed under PokemonsNotToTransfer segment regardless of KeepMin values

### Actual Behavior
Particular pokemons that I simply want to keep but do not meet KeepMin values are still being transferred

### Your config.json
```
"PokemonsNotToTransfer": [
    "lapras",
    "dragonite"
  ],
```

### Steps to Reproduce
"KeepMinCp": 1800,
"KeepMinIvPercentage": 85.0,
"KeepMinLvl": 6,
"KeepMinOperator": "or",
"UseKeepMinLvl": false,
"PrioritizeIvOverCp": true,

### Other Information
Version: 0.9.6
Windows Version: 8.1
What is affected (catch, transfer, evolve, recycle, snipe, powerup/levelup, other):
Transfer

**Comment by iwean** [comment]
Try to search down your config for string "PokemonsTransferFilter"
Use for individual transfer and overide your global transfer config.

**Comment by hardknoxni69a** [comment]
You will have to add to transfer filter because keepmin is higher in priority for bot so bot will follow your keep guideline before it does nottransfer values

**Comment by tws101** [comment]
I have found both the donottrasfer and the filter have been ignored in the current build...

My Post
https://github.com/NoxxDev/NecroBot/issues/675

**Comment by CreativiTimothy** [comment]
This is true. PokemonToTransfer and DoNoTransfer are both ignored in the current build 0.9.6
